site/app/views/CircuitPageView.php

```diff
@@ -151,6 +157,7 @@ public function getOutput($user, $config, $itemNavConfig) {
         }
 
         $return .= <<<HTML
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/jspdf/1.4.1/jspdf.debug.js" integrity="sha384-THVO/sM0mFD9h7dfSndI6TS0PgAGavwKvB5hAxRRvc0o9cPLohB0wb/PTA7LdUHs" crossorigin="anonymous"></script>
```

Comment:
Always put scripts from other sources in the <head> tag of the HTML doc

Reply:
done

Reply:
@Darokrithia 
Push the commit